flink-kubernetes-operator/src/test/java/org/apache/flink/kubernetes/operator/controller/RollbackTest.java

```diff
@@ -265,5 +265,28 @@ public void testRollback(
         assertTrue(reconciliationStatus.isLastReconciledSpecStable());
         assertEquals(ReconciliationState.DEPLOYED, reconciliationStatus.getState());
         assertNull(deployment.getStatus().getError());
+
+        if (deployment.getSpec().getJob() != null) {
+            deployment.getSpec().getJob().setState(JobState.SUSPENDED);
+            testController.reconcile(deployment, context);
+            Thread.sleep(500);
+            testController.reconcile(deployment, TestUtils.createEmptyContext());
+            testController.reconcile(deployment, context);
+            testController.reconcile(deployment, context);
```

Comment:
I think all the tests could still pass while removing these lines.

Reply:
good point this should not be necessary

Reply:
fixed